### PR TITLE
Run smoke tests with 64-bit VSCode

### PR DESCRIPTION
**Pull request by joyceerhl**
By default it used to download 32-bit.
Also ensure that we don't ship the images or docs directories.

-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [ ] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

**Review by DonJayamanne — APPROVED** [review]

File: .vscodeignore [hunk 1]
```diff
@@ -7,6 +7,8 @@
 !out/datascience-ui/viewers/commons.initial.bundle.js.map
 !out/datascience-ui/viewers/dataExplorer.js.map
 !out/datascience-ui/viewers/plotViewer.js.map
+images/**
```
Comment (IanMatthewHuff) [inline .vscodeignore:10]: What are these for?

Comment (IanMatthewHuff) [reply .vscodeignore:10]: Ahh, nevermind I see the comment.

**Review by IanMatthewHuff — APPROVED** [review]

File: .vscodeignore (hunk 1 above)
Comment (joyceerhl) [reply .vscodeignore:10]: We appear to be shipping https://github.com/microsoft/vscode-jupyter/tree/main/images I don't think we want that?

Comment (IanMatthewHuff) [reply .vscodeignore:10]: Yup, I just didn't see your initial commit comment on removal. Wasn't sure how it related to smoke tests. Makes sense now.

**Pull request merged**